topics/releases.md

```diff
@@ -1,81 +1,103 @@
-Redis release cycle
+Redis Release Cycle
 ===
 
-Redis is system software, and a type of system software that holds user
-data, so it is among the most critical pieces of a software stack.
+Redis is system software and a type of system software that holds user data, so it is among the most critical pieces of a software stack.
 
-For this reason our release cycle tries hard to make sure that a stable
-release is only released when it reaches a sufficiently high level of
-stability, even at the cost of a slower release cycle.
+For this reason, Redis' release cycle is such that it ensures highly-stable releases, even at the cost of slower cycles.
+
+New releases are published in the [Redis GitHub repository](http://github.com/redis/redis) and are also available for [download](/download).
+Announcements are sent to the [Redis mailing list](http://groups.google.com/group/redis-db) and by [@redisfeed on Twitter](https://twitter.com/redisfeed).
+
+Release Cycle
+---
 
 A given version of Redis can be at three different levels of stability:
 
-* unstable
-* development
-* frozen
-* release candidate
-* stable
+* Unstable
+* Release Candidate
+* Stable
 
-Unstable tree
-===
+### Unstable Tree
 
-The unstable version of Redis is always located in the `unstable` branch in
-the [Redis GitHub Repository](http://github.com/redis/redis).
+The unstable version of Redis is located in the `unstable` branch in the [Redis GitHub repository](http://github.com/redis/redis).
 
-This is the source tree where most of the new features are developed and
-is not considered to be production ready: it may contain critical bugs,
-not entirely ready features, and may be unstable.
+This branch is the source tree where most of the new features under development.
+`unstable` is not considered production-ready: it may contain critical bugs, incomplete features, and is potentially unstable features.
 
-However, we try hard to make sure that even the unstable branch is
-usable most of the time in a development environment without major
-issues.
+However, we try hard to make sure that even the unstable branch is usable most of the time in a development environment without significant issues.
 
-Forked, Frozen, Release candidate tree
-===
+### Release Candidate
 
-When a new version of Redis starts to be planned, the unstable branch
-(or sometimes the currently stable branch) is forked into a new
-branch that has the name of the target release.
+New minor and major versions of Redis begin as forks of the `unstable` branch.
+The forked branch's name is the target release
 
-For instance, when Redis 2.6 was released as stable, the `unstable` branch
-was forked into the `2.8` branch.
+For example, when Redis 6.0 was released as stable, the `unstable` branch was forked into the `6.2` branch. The new branch is the release candidate (RC) for that version.
```

Comment:
this is not technically correct. i think we can drop that sentence.
if not, then what it should say is that when 6.0 was made an RC, it was forked out of unstable, and unstable starts to work towards 6.2.

Reply:
Corrected